sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -424,7 +425,7 @@
     <dependency>
       <groupId>com.google.http-client</groupId>
       <artifactId>google-http-client</artifactId>
-      <version>${google-clients.version}</version>
```

Comment:
Problem was that google-api-client and google-http-client versions have diverged.

- The latest google-api-client version is 1.30.X.
- The latest google-http-client version is 1.34.X.